src/components/ModalPayment/ModalPayment.tsx

```diff
@@ -30,9 +30,9 @@ const ModalPayment = ({
   const payment: PaymentParams = {
     amount: Number(donatedAmt) * 100,
     currency: 'usd',
-    name: 'Gift Card',
+    item_type: purchaseType === 'donation' ? 'donation' : 'gift_card',
```

Comment:
Would be nice to abstract these into an enum on a class in the future. Don't need to do now

Reply:
ya can update later on